src/quiz.js

```diff
@@ -7,18 +7,22 @@ import UserService from "./service/UserService";
 
 const quizUi = new QuizUI(document);
 const wordService = new WordService();
-const userId = localStorage.getItem(CURR_USER_KEY);
+const userId = JSON.parse(localStorage.getItem(CURR_USER_KEY));
 const userService = new UserService();
+let wordsLeft;
+let totalWords;
+
+wordService.getTodayWordByUserId(userId).then((wordArray) => {
+  totalWords = wordArray.length;
+});
```

Comment:
this is async call, which means, it is not guaranteed you will get its value before your run into the next lines (which may depend on this value being available)